docs/language/composite-types.mdx

```diff
@@ -917,15 +917,6 @@ use(resource: <-res)
 res.value
 ```
 
-```cadence
```

Comment:
Is there any reason for removing this? This example is explicitly demonstrating that resources cannot be lost

Reply:
As the respective commit message explains, [there are currently](https://docs.onflow.org/cadence/language/composite-types/#resources) two `forgetToUse` examples that demonstrate the same principle. I remove one of them to avoid confusion/duplication.

Reply:
Ah, good catch, I didn't see thee commit message 👍